src/main/kotlin/com/thelastpickle/tlpstress/Metrics.kt

```diff
@@ -1,19 +1,17 @@
 package com.thelastpickle.tlpstress
 
 import com.codahale.metrics.MetricRegistry
+import com.codahale.metrics.ScheduledReporter
 import java.util.concurrent.TimeUnit
 
-class Metrics {
-    val metrics = MetricRegistry()
-
-    val reporter = SingleLineConsoleReporter(metrics)
+class Metrics(metricRegistry: MetricRegistry, val reporter: ScheduledReporter) {
```

Comment:
I'm not wild about restricting this to one reporter, b/c of the reason I listed above (I don't like disabling console reporter implicitly).   I like the idea behind the design though.  If we're going to do this change now, let's make it take a list.  There's a reporter I had in mind for later I'd like to set things up for.  Mind making this take a list of reporters, and calling `.start()` on all of them?

Reply:
As noted, we're now using a list of reporters.